interface/src/app/about/about.tpl.html

```diff
@@ -13,13 +13,24 @@ <h3>About ADAGE</h3>
   and
   <a href="http://biorxiv.org/content/early/2017/04/10/078659">eADAGE</a>;
   however, the server was designed to be general and can be used for other
-  types of machine learning models as well.
+  types of machine learning models as well. Signatures from PCA, ICA, NMF, and
+  other factorization methods can also be loaded into the server.
+</p>
+
+<p>
+  The ADAGE server is open source. If you'd like to start your own instance of
+  the ADAGE server, modify the source, or participate in development, please
+  visit our
+  <a href="https://github.com/greenelab/adage-server">
+    GitHub repository
+  </a>.
```

Comment:
Really small thing - the way this is presented in HTML now makes an empty space appear between `repository` and the `.` period. Is there a way we can get rid of this? shall we just put it all in one line here:
`<a href="https://github.com/greenelab/adage-server">GitHub repository</a>.`? It looks like it is 77 characters long, including the 2 spaces of leading indentation.

Reply:
Let me try that.